src/scss/feed.css

```diff
@@ -0,0 +1,19 @@
+body {
+  background: #654ea3;
+  background: linear-gradient(to right, #e83574, #68034b);
+  min-height: 100vh;
```

Comment:
This could be achieved with one of the bootstrap sizing classes `min-vh-100`. See this doc https://getbootstrap.com/docs/5.3/utilities/sizing/#relative-to-the-parent